cma/evolution_strategy.py

```diff
@@ -4591,6 +4602,13 @@ def fmin_con(objective_function, x0, sigma0,
     True
     >>> # al.logger.plot()  # plots the evolution of AL coefficients
 
+    >>> x, es = cma.evolution_strategy.fmin_con(
+    ...             cma.ff.sphere, 2 * [0], 1, g=lambda x: [y+1 for y in x], post_optimization=True,
+    ...             options={'seed': 1, 'verbose': -9})
+
+    >>> hasattr(es, 'best_feasible_post_opt')
```

Comment:
We should assert here that the solution is feasible, like `assert all(y <= -1 for y in x)`.